revolv/revolv_cms/models.py

```diff
@@ -212,3 +212,60 @@ class PaymentModalSettings(BaseSetting):
     content_panels = [
         FieldPanel('payment_modal_paragraph', classname="full"),
     ]
+
+@register_setting
+class DashboardSettings(BaseSetting):
+    """Editable settings for the dashboard page."""
+    
+    impact_statistics_header_text = models.CharField(
+        max_length = 100,
+        help_text = "The heading above the statistics in the 'My Impact' section of the dashboard, e.g. 'Thank you for contributing! Your contribution has...'.",
+        default= 'Thank you for contributing! Your contribution has...'
+    )
+
+    repayment_statistic_top_description_text = models.CharField(
+        max_length = 20,
+        help_text = "The top section of the description of the repayments statistic on the 'My Impact' section of the dashboard.",
```

Comment:
Do you think you could elaborate in this help text about how there are two editable sections for each statistic? This help text is really important because the administrators will see it, and they need to make sure they know intuitively what they're editing. In fact, I think it might make sense to put these statistic fields in another `DashboardImpactSettings` setting, because the admins see all the fields on one page, and this is a lot for the dashboard settings page, especially with the category preference stuff which is hidden down at the bottom.